react-components/src/apps/listing/AppGridListing.js

```diff
@@ -279,7 +280,7 @@ AppGridListing.defaultProps = {
 
 AppGridListing.propTypes = {
     parentId: PropTypes.string.isRequired,
-    apps: PropTypes.array.isRequired,
+    apps: PropTypes.array,
```

Comment:
Did we not want this required?  I think in other components, I've initialized my props with an empty array instead of null in order to get around the warning (not sure if that's the case here, but thought I'd mention it).

Reply:
🤔 The `apps` props in my case is a `splittable` and I had to initialize to null.